<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDoctorEdit</class>
 <widget class="QDialog" name="dlgDoctorEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblLicence">
       <property name="text">
        <string>Licence :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledLicence"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblData">
       <property name="text">
        <string>Additional information :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="ptData"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkIndependent">
     <property name="text">
      <string>Independent from the local studio.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgDoctorEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgDoctorEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
